docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.9'
+services:
+  web:
+    build: /Users/noname/PycharmProjects/game_nerd
```

Comment:
This should be relative path(since I don't have these folders in my PC). So that I'd be able to clone this repo, and use it straight away (same for volume below)
Will something like this work?

```
    build:
        context: .
        dockerfile: Dockerfile
  ```

Reply:
So, in last commit changes were implemented accordingly.